bindings/wasm/identity_wasm/examples/README.md

```diff
@@ -13,7 +13,7 @@ Examples can be run against
 - or an existing network, e.g. the IOTA testnet
 
 When setting up the local node, you'll also need to publish an identity package as described in
-[Getting Started](../../../../README.md#getting-started).
+[Local Network Setup](https://docs.iota.org/iota-identity/getting-started/local-network-setup) in the wiki.
```

Comment:
since we point to the docs for this content, should we remove it from the repo?

Reply:
Ah, yes. The link was also pointing to a non-existing header with the content being removed. >.<